### DISCUSSION TITLE
Configured and still 0 packages

### DISCUSSION THREAD

**Discussion by nytram-md** [moralmunky/Home-Assistant-Mail-And-Packages#572] [Q&A]
![Screenshot_2021-11-16-07-17-46-302_io homeassistant companion android](https://user-images.githubusercontent.com/55548608/141940618-7af02561-d5bf-40b2-8e8e-319fea570362.jpg)
![Screenshot_2021-11-16-07-17-59-785_io homeassistant companion android](https://user-images.githubusercontent.com/55548608/141940635-231ecdd9-4e40-4a2c-a81d-cbe0f0859707.jpg)

I have about 5 orders with amazon.co.uk coming but still showing as 0 packages everywhere.

Is there anything I can check please ?

I used SMTP.live.com for Hotmail email. I have tried SMTP.outlook.com too.

Thank you

**Comment by firstof9** [comment]
[Covered in the wiki](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki/Configuration-and-Email-Settings#hotmailoutlooklive).
